--- a/sglang/srt/models/qwen2.py
+++ b/sglang/srt/models/qwen2.py
@@ -52,7 +52,7 @@ from sglang.srt.model_loader.weight_utils import (
 )
 from sglang.srt.server_args import get_global_server_args
 from sglang.srt.utils import add_prefix, make_layers,is_dcu,get_bool_env_var
-# from sglang.srt.utils.hf_transformers_utils import get_rope_config
+from sglang.srt.utils.hf_transformers_utils import get_rope_config
 _is_dcu = is_dcu()
 if _is_dcu:
     from lightop import split_qkv_rms_rotary_embedding_fuse_with_kv_store_quant
